producer/event_producer.py

```diff
@@ -164,7 +164,7 @@ def pgoutput_msg_processor(self, msg):
             logger.debug(f'Table name: {table_name}')
             logger.debug(f'Schema: {schema}')
 
-            if table_name in self.__pg_tables:
+            if table_name in self.__pg_tables or self.__pg_tables == 'public.*':
```

Comment:
or '.*' in self.__pg_tables (it may nor necessarily be public schema)

Reply:
done